frontend/src/components/Following/FollowingButton.js

```diff
@@ -0,0 +1,67 @@
+import '../../config';
+import React, { useState, useEffect } from 'react';
+import axios from "axios";
+
+// Stylesheet
+import './FollowingButton.scss'
+
+export default function FollowingButton(props)
+{
+    const [isFollowing, setFollowing] = useState(false);
```

Comment:
I know this is picky, but shouldn't it be `setIsFollowing`?

Reply:
Yes it should! Thanks :)